src/main/java/com/startsteps/ecommerceapi/utils/Default.java

```diff
@@ -0,0 +1,11 @@
+package com.startsteps.ecommerceapi.utils;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.CONSTRUCTOR)
+@Retention(RetentionPolicy.CLASS)
+public @interface Default {
```

Comment:
is this needed?

Reply:
without custom default annotation mapstruct has conflicts with multiple constructors :(